interview-questions.md

```diff
@@ -6,41 +6,54 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
+
+
+
+
+
 1. What is the difference between a parameter and an argument?
 
 Your answer:
+the best way i can explain an argument is using an arugument is  {const example = (parameter )=>{
+return( argument 1, argument 2 )
+}}
 
 Researched answer:
+The values that are declared within a function when the function is called are known as an argument. Whereas, the variables that are defined when the function is declared are known as a parameter.
+
```

Comment:
A parameter can be thought of as a placeholder until the function is invoked along with an argument, or the data that will be passed to the function.